### PR TITLE
add cargo total capacity

### PR DESCRIPTION
**Pull request by d-v-p**
Hello,

I found it useful to see total cargo capacity on cargo page.
I implemented it for myself and decided to create a pull request, just in case you will find it useful too.

### CODE REVIEW

File: edreader/cargo.go [hunk 1]
```diff
@@ -72,7 +72,8 @@ func handleCargoFile(file string) {
 }
 
 func renderCargo(page *mfd.Page, cargo Cargo) {
-	page.Add("#  Cargo: %3d  #", cargo.Count)
+	page.Add("#    Cargo     #")
```
Comment (peterbn) [inline edreader/cargo.go]: Could you get it into one line?
```
page.Add("#Cargo: %03d/%03d#")
```
_should_ render as 16 characters if I'm counting right.

That way you have to scroll one line less. But, if you think it looks too cramped that way, then just let me know.

File: edreader/modulesInfo.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	Slot        string
+	Item        string
+}
+
+var currentModules ModulesInfo
+
+func handleModulesInfoFile(file string) {
+	data, err := ioutil.ReadFile(file)
+	if err != nil {
+		fmt.Println(err)
```
Comment (peterbn) [inline edreader/modulesInfo.go]: Please use
```
import (log "github.com/sirupsen/logrus")
```
and 
```
log.Errorln(err)
``` 
to match error logging in the rest of the project.

I see that I've used this same check in `cargo.go`, but that one should also be using the logging framework, I just missed it when I switched my error logging.

**Review by peterbn — CHANGES_REQUESTED** [review]
Hi Dmitriy.

Thank you very much - this is the first ever pull request I've received, so I'm very flattered that you took the time to write it up.

I have two minor comments. Please let me know if you've already tried the one-line version of cargo capacity and rejected it as being too cramped.

File: edreader/cargo.go (hunk 1 above)
Comment (d-v-p) [reply edreader/cargo.go]: I added spaces after "#" character while I was testing and line was scrolling
Actually page.Add("#Cargo: %03d/%03d#") looks good, I will add changes

File: edreader/modulesInfo.go (hunk 2 above)
Comment (d-v-p) [reply edreader/modulesInfo.go]: Ok, I will update it

**Comment by d-v-p** [comment]
Hello Peter,

I added changes. Please take a look when you have a moment.

**Review by peterbn — COMMENTED** [review]
Thank you very much. Merged.

**Pull request merged**